test/SILOptimizer/floating_point_conversion.swift

```diff
@@ -1,12 +1,36 @@
 // RUN: %target-swift-frontend -O -module-name=test -emit-sil %s | %FileCheck %s
 
 func convert<
-   T: BinaryFloatingPoint, U: BinaryFloatingPoint
- >(_ value: T, to: U.Type) -> U {
-   U(value)
+  T: BinaryFloatingPoint, U: BinaryFloatingPoint
+>(_ value: T, to: U.Type) -> U {
+  U(value)
 }
 
-// Check that the follwing functions can be optimized to no-ops.
+// Check that the following functions can be optimized to concrete conversions.
+
+// CHECK-LABEL: sil @$s4test0A13DoubleToFloatySfSdF
+// CHECK:      bb0(%0 : $Double):
+// CHECK:        struct_extract %0 : $Double, #Double._value
```

Comment:
The usual way to check this is to use patterns for the values, e.g.
```
[[ARG:%[0-9]+]] = struct_extract %0
[[CONV:%[0-9]+]] = builtin "fptrunc_FPIEEE64_FPIEEE32"([[ARG]])
...
```
But doing it with CHECK-NEXT is perfectly fine for this test.